packages/app/routes/api/users.js

```diff
@@ -200,7 +200,7 @@ router.get(
       })
       let { projects } = user
       projects = projects.sort((a, b) =>
-        a.createdAt > b.createdAt ? 1 : b.createdAt > a.createdAt ? -1 : 0
+        a.createdAt < b.createdAt ? 1 : b.createdAt < a.createdAt ? -1 : 0
```

Comment:
There are keywords to deal with sort dates. See the Mongoose docs -> https://mongoosejs.com/docs/api.html#query_Query-sort